use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -124,35 +143,28 @@ EOF
 $ sudo clr-boot-manager update
 ```
 
-### Replace Clear Linux kernel with LTS kernel
+### Replace Clear Linux kernel with LTS kernel (Optional)
+
+If you plan to replace the host driver then the LTS kernel is necessary. This
+installs the Linux LTS kernel version `4.19` from 2018 because the 
+crypto modules are set to `m` and not `y`, which allows you to replace them 
+with the out-of-tree driver. Otherwise you need to build your own Clear Linux 
+kernel with the QAT drivers compiled in. 
 
 ```sh
+$ sudo swupd bundle-add kernel-lts2018 linux-lts-dev
 $ lts_kernel=$(sudo clr-boot-manager list-kernels | grep lts2018 | head -n 1)
 $ sudo clr-boot-manager set-kernel $lts_kernel
 $ sudo reboot
 ```
 
-### Download Intel QAT drivers
+### Compile Intel QAT drivers against host kernel and install (Optional)
 
-This will download the Intel QAT drivers from [`01.org`](https://01.org/intel-quickassist-technology). 
-Make sure to check the website for the latest version.
-
-```sh
-$ mkdir -p $QAT_SRC
-$ cd $QAT_SRC
-$ curl -O $QAT_DRIVER_URL
-$ QAT_DRIVER_NAME=$(echo $QAT_DRIVER_URL | rev | cut -d"/" -f1 | rev)
-$ tar xzvf $QAT_DRIVER_NAME
-```
-
-### Compile Intel QAT drivers against host kernel 
-
-Now, the Linux kernel headers bundle `linux-dev` is present. This replaces any
-existing modules provided by Clear Linux. If the Clear Linux kernel updates,
-you must rebuild and reinstall these modules against the kernel. By doing it
-this way you no longer have to rely on your distribution to include the QAT
-firmware. These modules will load at every reboot and add the `qat_service`
-helper script.
+This replaces any existing modules provided by Clear Linux. If the Clear Linux
+kernel updates, you must rebuild and reinstall these modules against the 
```

Comment:
Can we help the user here any more by providing details of how they'd know the kernel has been updated?

Reply:
I don't know of a tool or easy way to test this. I think the docker container below just wouldn't work. I am open to suggestions.

Reply:
I decided to remove this section after Mikko's feedback.